bitnami/redis-cluster/templates/_helpers.tpl

```diff
@@ -0,0 +1,336 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "redis.name" -}}
```

Comment:
Change all the helpers from `redis.` to `redis-cluster.`

There may be an umbrella chart that may want to use both redis and redis-cluster charts. In order to avoid redefinition of the helpers, use a different name